src/shared/constants/field-validators/field-validators.ts

```diff
@@ -50,3 +50,10 @@ export const getPassCertificateAMOD1Validator = (): FieldValidators => {
     maxByteLength: 8,
   };
 };
+
+export const getDL196CBTCertificateNumberValidator = (): FieldValidators => {
+  return {
+    pattern: /^[1-9][0-9]{6,6}$/g,
```

Comment:
Pattern should be simplified to:

/^[1-9][0-9]{6}$/g

Reply:
just tested, seems the extra 6 doesnt do anything.